pkg/restore/restic_restore_action.go

```diff
@@ -36,10 +36,14 @@ type ResticRestoreAction struct {
 	initContainerImage string
 }
 
-func NewResticRestoreAction(logger logrus.FieldLogger) *ResticRestoreAction {
+func NewResticRestoreAction(logger logrus.FieldLogger, initContainerImageName string) *ResticRestoreAction {
+	if initContainerImageName == "" {
```

Comment:
I'll have to think a bit more on how to accomplish this, but it'd be awesome if the `initContainerImage` function could just return the image from the args.